lib/article_status_manager.rb

```diff
@@ -48,11 +48,7 @@ def update_status(articles)
     # This happens in situations such as history merges.
     # If articles move in between title/namespace updates and mw_page_id updates,
     # then it's possible to end up with inconsistent data.
-    update_article_ids @deleted_page_ids
```

Comment:
I forget... what's the context for the PR? Note that this file does a whole lot of fiddly things to ensure as consistent state as possible between the Dashboard and Wikipedia, including handling for deletion, undeletion, and ID changes on the Wikipedia side (which can happen in various situations that we don't try to model on the Dashboard side, such as when the history of two pages get merged into one article).

Reply:
Yes, for now I have simply commented that out but I plan on calling this function from my worker as well.

The context is a `FIXME` which says that instead of relying on Replica, an article should be marked deleted if there's a deletion log.

Reply:
cool. this general approach looks good.